src/System.Net.Security/tests/FunctionalTests/CertificateValidationClientServer.cs

```diff
@@ -21,11 +21,11 @@ public class CertificateValidationClientServer : IDisposable
 
         public CertificateValidationClientServer()
         {
-            _serverCertificateCollection = Configuration.Certificates.GetServerCertificateCollection();
-            _serverCertificate = Configuration.Certificates.GetServerCertificate();
+            _serverCertificateCollection = System.Net.Test.Common.Configuration.Certificates.GetServerCertificateCollection();
```

Comment:
Did you try building outside VS? I don't really understand why this would fail. In any case, if the using doesn't work, probably a better thing would be to add an alias at the top so that you don't have to type the fully qualified name everytime.

---
In reply to: [81846153](https://github.com/dotnet/corefx/pull/12351#discussion_r81846153) [](ancestors = 81846153)